Fix documentation, grammar, and spelling in bindings Q-S (#19586)
openhab/openhab-addons: bundles/org.openhab.binding.silvercrestwifisocket/README.md

# Silvercrest Wi-Fi Plug Binding

> This binding integrates the Silvercrest WiFi Socket SWS-A1 sold by Lidl and the EasyHome WiFi Socket DIS-124 sold by Aldi.

Reviewer: ```suggestion
This binding integrates the Silvercrest Wi-Fi Socket SWS-A1 sold by Lidl and the EasyHome Wi-Fi Socket DIS-124 sold by Aldi.
```

(same as https://github.com/openhab/openhab-addons/pull/19586#discussion_r2651250033)

Author: done... all

## Supported Things

- Silvercrest WiFi Socket SWS-A1 - [(Owner Manual)](https://www.lidl-service.com/static/118127777/103043_FI.pdf)   --   Tested with firmware version: 1.41, 1.60, 1.70
- EasyHome WiFi Socket DIS-124 <https://www.aldi-sued.de/de/infos/aldi-sued-a-bis-z/s/serviceportal/ergebnisliste/sis/si/wifi-steckdose/>

## Discovery

The discovery of WiFi Sockets is always running in the background.
If a command is sent to the WiFi socket using the Android/iOS app or if the physical button on the device is pressed, the device will be recognized and placed in the Inbox.

## Binding Configuration

The binding does not require any special configuration.
The WiFi Socket should be connected to the same WiFi network.

## Thing Configuration

To configure a WiFi Socket manually, the MAC address and the vendor are required.
You can check the WiFi Socket MAC address in your router or using a mobile app.
Supported vendors are either Silvercrest (Lidl) or EasyHome (Aldi).

> WiFi Socket Thing parameters:

Author: ```suggestion
Wi-Fi Socket Thing parameters:
```

## Channels

> The Silvercrest WiFi Socket supports the following channel:

Author: ```suggestion
The Silvercrest Wi-Fi Socket supports the following channel:
```

| Channel Type ID | Item Type | Description         |
|-----------------|-----------|---------------------|

> | switch          | Switch    | WiFi Socket Switch. |

Author: ```suggestion
| switch          | Switch    | Wi-Fi Socket Switch. |
```
